apps/codelab/tsconfig.json

```diff
@@ -1,9 +1,32 @@
 {
-  "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "types": ["jasmine", "node"]
+    "target": "es2022",
```

Comment:
should this match whatever we have in browserconfig? (i think we have es6 there)

Reply:
browserconfig isn't only for ts, and for styles too. Currently using this target
https://github.com/angular/angular-cli/blob/main/packages/schematics/angular/workspace/files/tsconfig.json.template

But I agree, in the future we need to sync browserlist with ts target

Reply:
ah, good point